src/shared/database/setup.ts

```diff
@@ -121,6 +121,7 @@ async function setupValidatorsTable(): Promise<void> {
       table.string('unl')
       table.string('domain')
       table.boolean('domain_verified')
+      table.string('server_version')
```

Comment:
We'll need to carefully deploy this to staging to see if this will correctly add this column. 

If not we'll have to manually go and add the column

Reply:
This won't add the column, unless the table doesn't exist (obviously it already does in staging and prod). This is how to modify a column if the table already exists: https://github.com/ripple/validator-history-service/blob/main/src/shared/database/setup.ts#L142-#L145 (you could probably figure out how to modify this code to get it to add a column instead)

Reply:
I added the column in the if statement on line 131.